streams/th.m3u

```diff
@@ -9,10 +9,10 @@ https://thaipbs-ujxrch.cdn.byteark.com/live/playlist.m3u8
 https://thaipbs-ujxrch.cdn.byteark.com/live/playlist_1080p/index.m3u8
 #EXTINF:-1 tvg-id="Channel5.th" status="online",Channel 5 (480p)
 https://tc-live1.sanook.com/live/22302_ch5.m3u8
-#EXTINF:-1 tvg-id="Channel5.th" status="timeout",Channel 5 (1080p)
-http://110.170.117.27:1935/apptv5hd1live/vdo-tv5hd1/playlist.m3u8
-#EXTINF:-1 tvg-id="Channel7.th" status="blocked",Channel 7 (240p)
-https://bcovlive-a.akamaihd.net/2d37038b355f4ea6a6b0d46993dc285c/ap-southeast-1/5282994675001/profile_0/chunklist.m3u8
+#EXTINF:-1 tvg-id="Channel5.th" status="online",Channel 5 (1080p)
+https://639bc5877c5fe.streamlock.net/tv5hdlive/tv5hdlive/chunklist_w555179815.m3u8
```

Comment:
Change the chunklist part and write playlist instead

Reply:
Thank you ^^